src/main/java/com/github/packageurl/PackageURLBuilder.java

```diff
@@ -272,10 +288,7 @@ public String getSubpath() {
      * @return all qualifiers set in this builder, or an empty map if none are set
      */
     public Map<String, String> getQualifiers() {
-        if (qualifiers == null) {
-            return null;
-        }
-        return Collections.unmodifiableMap(qualifiers);
+        return qualifiers != null ? Collections.unmodifiableMap(qualifiers) : Collections.emptyMap();
```

Comment:
The current Javadoc suggests that this method returns an empty map instead of `null`, so I changed the implementation…